### PR TITLE
[Week 7] TP Iterations 4: PPP

### LINKED ISSUE #11: [Week 7] TP Iterations 4: PPP
https://nus-cs2103-ay2324s1.github.io/website/schedule/week7/project.html#4-add-a-skeletal-ppp

Once we have all done this, this issue can be closed.

---

- [x] @bwangpj (#42)
- [x] @Cloud7050 (#34)
- [x] @mamayuan (#44)
- [x] @rayshawntan (#49)

Click the checkbox directly, and the post should automatically get edited to mark off the list item.

### PR DESCRIPTION
**Pull request by rayshawntan**
Add my skeletal PPP.

Closes #11.

### CODE REVIEW

File: docs/team/rayshawntan.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: mamayuan's Project Portfolio Page
```
Comment (mamayuan) [inline docs/team/rayshawntan.md:3]: lol remember to change the title

**Review by mamayuan — APPROVED** [review]

**Review by Cloud7050 — CHANGES_REQUESTED** [review]
As per Ma Yuan's comment.

File: docs/team/rayshawntan.md (hunk 1 above)
Comment (rayshawntan) [reply docs/team/rayshawntan.md:3]: oops, my bad

**Comment by Cloud7050** [comment]
Btw can we also delete johndoe.md for PPP and johndoe.png for AboutUs?

**Comment by rayshawntan** [comment]
I've deleted the johndoe.md in the previous commit. johndoe.png is deleted in #48

**Review by bwangpj — APPROVED** [review]
Lgtm!

**Pull request merged**